api/libproject/devstart.rb

```diff
@@ -259,6 +278,16 @@ def connect_to_cloud_db(*args)
   })
 end
 
+def update_cloud_config(*args)
+  run_with_cloud_sql_proxy(args, "connect-to-cloud-db", lambda { |project, account, creds_file|
+    read_db_vars(project)
+    ENV["DB_PORT"] = "3307"
+    unless system("cd tools && ../gradlew --info loadConfig && cd ..")
```

Comment:
What's the difference between common.run_inline and system()?

Reply:
run_inline (based on my experimentation) only lets you run a single command. If you want to chain together commands like this, you need to use system or back quotes.

Reply:
There is a `pipe` function for piping. Another option for this particular command would be:
```ruby
Dir.chdir("tools") do
  common.run_inline %W{../gradlew --info loadConfig}
end
```